packages/plugins/plugin-facebook-app-events/src/methods/screen.ts

```diff
@@ -0,0 +1,25 @@
+import type { ScreenEventType } from '@segment/analytics-react-native';
+import { AppEventsLogger } from 'react-native-fbsdk-next';
+
+// FB Event Names must be <= 40 characters
+// 'Viewed' and 'Screen' with spaces take up 14
+const MAX_CHARACTERS_EVENT_NAME = 26;
```

Comment:
nit: How about putting it like this so that if we change it it is all automatically handled:
```suggestion
const PREFIX = 'Viewed ';
const SUFFIX = ' Screen';
const MAX_CHARACTERS_EVENT_NAME = 40 - PREFIX.length - SUFFIX.length;
```
Down below you can return the sanitizedName as:
```
return `${PREFIX}${trimmedName}${SUFFIX}`;
```